src/coreclr/src/System.Private.CoreLib/src/System/Reflection/RuntimeModule.cs

```diff
@@ -211,11 +222,11 @@ public override byte[] ResolveSignature(int metadataToken)
                     declaringType = (RuntimeType)ResolveType(tkDeclaringType, genericTypeArguments, genericMethodArguments);
                 }
 
-                return System.RuntimeType.GetFieldInfo(declaringType, fieldHandle);
+                return RuntimeType.GetFieldInfo(declaringType, fieldHandle);
             }
             catch (MissingFieldException)
             {
-                return ResolveLiteralField(tk, genericTypeArguments, genericMethodArguments);
+                return ResolveLiteralField(new MetadataToken(metadataToken), genericTypeArguments, genericMethodArguments);
```

Comment:
```suggestion
                return ResolveLiteralField(metadataToken, genericTypeArguments, genericMethodArguments);
```
ResolveLiteralField takes int. We should not the roundtrip via struct MetadataToken here. (MetadataToken has implicit conversion to int.)